.github/workflows/django_testing_ci.yml

```diff
@@ -52,12 +52,12 @@ jobs:
       uses: actions/cache@v3
       with:
         path: ~/venv
-        key: ${{ runner.os }}-python-packages-${{ hashFiles('requirements.txt') }}
+        key: ${{ runner.os }}-python310-packages-${{ hashFiles('requirements.txt') }}
```

Comment:
Is this distinction needed in the long run?

Reply:
I think it's needed as long as there's 3.8 workflows running in the repository